src/kiwixapp.cpp

```diff
@@ -113,6 +114,28 @@ void KiwixApp::init()
             m_library.asyncUpdateFromDir(dir);
         }
     }
+
+    /* Place session file in our global library path */
+    QDir dir(m_libraryDirectory);
+    this->mp_session = new QSettings(std::move(dir).filePath("user.session"),
```

Comment:
Why is `std::move()` needed?

Reply:
Oops forgot to remove the std::move. No point moving since I am not performing any copying.

Reply:
Done